mods/gen3/random-teams.js

```diff
@@ -385,38 +385,33 @@ class RandomGen3Teams extends RandomGen4Teams {
 			item = 'Stick';
 		} else if (template.species === 'Marowak') {
 			item = 'Thick Club';
-		} else if (template.species === 'Pikachu') {
-			item = 'Light Ball';
 		} else if (template.species === 'Shedinja') {
 			item = 'Lum Berry';
-		} else if (template.species === 'Unown') {
-			item = 'Twisted Spoon';
-		} else if (template.species === 'Wobbuffet') {
-			item = ['Leftovers', 'Sitrus Berry'][this.random(2)];
 		} else if (template.species === 'Slaking') {
 			item = 'Choice Band';
+		} else if (template.species === 'Unown') {
+			item = 'Twisted Spoon';
 		} else if (hasMove['trick']) {
 			item = 'Choice Band';
 		} else if (hasMove['bellydrum']) {
-			item = 'Sitrus Berry';
+			item = 'Salac Berry';
 		} else if (hasMove['rest'] && !hasMove['sleeptalk'] && ability !== 'Natural Cure' && ability !== 'Shed Skin') {
 			item = 'Chesto Berry';
 
 		// Medium priority
-		} else if (hasMove['endeavor'] || hasMove['flail'] || hasMove['reversal'] || hasMove['endure']) {
-			if (template.baseStats.spe < 108) {
+		} else if (hasMove['leechseed']) {
+			item = 'Leftovers';
+		} else if (hasMove['endeavor'] || hasMove['flail'] || hasMove['reversal'] || hasMove['endure'] ||
+			hasMove['substitute'] && counter.Status < 3 && template.baseStats.hp + template.baseStats.def + template.baseStats.spd < 250 && this.random(2)) {
+			if (template.baseStats.spe <= 90 && !counter['speedsetup']) {
 				item = 'Salac Berry';
-			} else if (counter.Physical > counter.Special) {
+			} else if (counter.Physical >= counter.Special) {
 				item = 'Liechi Berry';
 			} else {
 				item = 'Petaya Berry';
 			}
```

Comment:
Having a choice band and special move dosen't make much sense to me, though with the type split in gen 3 and the way the format is setup, this could work.

Reply:
Since there's no consistent item to boost SpA, it can be worth it to have CB even with a special attack since it still boosts 3/4 of your moves.

Some examples from Smogon analyses (and it should be even more useful in Randbats that OU since the EVs aren't specialized):
http://www.smogon.com/dex/rs/pokemon/flygon/
http://www.smogon.com/dex/rs/pokemon/gyarados/
http://www.smogon.com/dex/rs/pokemon/salamence/

Reply:
sounds good